dashboard/app/models/resource.rb

```diff
@@ -17,8 +17,21 @@
 
 # A Resource represents a link to external material for a lesson
 # #
-# @attr [String] name - The user-visisble name of the resource
```

Comment:
It looks like when you added the documentation for the new things you might have deleted the documentation for the old things?

Reply:
I realized that in the other examples of the YARD docs it only documented the attributes. I'm not sure what the correct way to use this documentation style is though....

Reply:
@davidsbailey @uponthesun Thoughts on how this is supposed to be used?

Reply:
I like including both things defined in columns and things defined in `serialized_attrs` as `@attr`. Not sure what the official YARD stance on that, but I think it's more important to just have the info we need in there, regardless.

Reply:
I would vote for documenting db columns as well as attributes. probably just none of us caught it that we had overlooked those earlier.

Reply:
Sounds good! I'll add them back in!

FWIW, it was the `attr` part of the documentation that suggested it might be documentation specifically for `serialized_attrs`.